packages/module/src/Message/Message.tsx

```diff
@@ -185,6 +185,10 @@ export interface MessageProps extends Omit<HTMLProps<HTMLDivElement>, 'role'> {
   editFormProps?: FormProps;
   /** Sets message to compact styling. */
   isCompact?: boolean;
+  /** Disables markdown parsing for message, allowing only text input */
+  disableMarkdown?: boolean;
```

Comment:
Non-blockers:

- Would we want to include some verbiage in the description for when you may want to disable markdown?
- react-core conventions speaking: `isMarkdownDisabled` instead for the prop name?

Reply:
Updated prop name! Let's wait on the verbiage for now and update in a little bit if that's ok.